internal/cli/cmd/cluster/create_engine.go

```diff
@@ -0,0 +1,255 @@
+/*
```

Comment:
"engine" is DBMS specific term, which in inappropriate for cluster like Kafka.

Reply:
rename to `ClusterType`.